tests/Umbraco.Tests.UnitTests/Umbraco.Cms.Api.Management/Services/Signs/HasCollectionSignProviderTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using NUnit.Framework;
+using Umbraco.Cms.Api.Management.Services.Signs;
+using Umbraco.Cms.Api.Management.ViewModels;
+using Umbraco.Cms.Api.Management.ViewModels.DocumentType;
+using Umbraco.Cms.Api.Management.ViewModels.Tree;
+
```

Comment:
Could you just clean up these extra line-breaks when you see them please - there are a few in different classes.  Not a big deal but just to keep things tidy we just have one line between code structures.